src/EntryPoints/Specials/SpecialManageApprovers.php

```diff
@@ -35,7 +35,7 @@ public function execute( $subPage ): void {
 
 		$request = $this->getRequest();
 		if ( $request->wasPosted() ) {
-			$this->handlePostRequest( $request, $approversCategories );
```

Comment:
Looks like `approversCategories` is no longer used at all?

Reply:
Yes, After @malberts 's PR, it's cleaner to directly `approverRepository` call because it is available as a dependency.

https://github.com/ProfessionalWiki/PageApprovals/pull/95